test/e2e/auth/BUILD

```diff
@@ -0,0 +1,49 @@
+package(default_visibility = ["//visibility:public"])
+
+licenses(["notice"])
+
+load(
+    "@io_bazel_rules_go//go:def.bzl",
+    "go_library",
+)
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "apparmor.go",
```

Comment:
apparmor was already moved into sig-node. I don't have a strong preference for which sig owns it, as I'm the owner and part of both sigs. We should probably decide which sig is ultimately responsible for the pod SecurityContext and PodSecurityPolicy, although obviously there's a lot of overlap.

@kubernetes/sig-node-pr-reviews